Source/OxyPlot.SkiaSharp/SkiaRenderContext.cs

```diff
@@ -338,7 +338,7 @@ public void DrawRectangle(OxyRect rectangle, OxyColor fill, OxyColor stroke, dou
         }
 
         /// <inheritdoc/>
-        public void DrawRectangles(IList<OxyRect> rectangles, OxyColor fill, OxyColor stroke, double thickness, EdgeRenderingMode edgeRenderingMode)
+        public void DrawRectangles(IList<OxyRect> rectangles, OxyColor fill, OxyColor stroke, double thickness, EdgeRenderingMode edgeRenderingMode, double[] dashArray = null)
```

Comment:
Unused parameter (issue throughout file)

Reply:
What would your expectation to solve this be here to solve this? 

Add a new specific method on `IRenderContext` and use it appropriately.